src/converse-sendbutton.js

```diff
@@ -0,0 +1,94 @@
+// Converse.js (A browser based XMPP chat client)
+// http://conversejs.org
+//
+// Copyright (c) 2012-2017, Jan-Carel Brand <jc@opkode.com>
+// Licensed under the Mozilla Public License (MPLv2)
+//
+/*global Backbone */
+
+/* This plugin is to add an option of to have send button in chat box */
+
+(function(root, factory) {
+    define([
+        "converse-core",
+        "converse-chatview",
+        "converse-muc",
+        "converse-headline"
+    ], factory);
+}(this, function(converse) {
+    "use strict";
+    var $ = converse.env.jQuery;
+
+    converse.plugins.add('converse-sendbutton', {
+
+        overrides: {
+            // Overrides mentioned here will be picked up by converse.js's
+            // plugin architecture they will replace existing methods on the
+            // relevant objects or classes.
+            //
+            // New functions which don't exist yet can also be added.
+
+            ChatBoxView: {
+                events: {
+                    'click .send-button': 'onSendButtonClicked'
+                },
+
+                onSendButtonClicked: function(ev) {
+                    /* Event handler for when a send button is clicked in a chat box textarea.
+                     */
+                    var _converse = this.__super__._converse;
+                    ev.preventDefault();
+                    var textarea = $(ev.target).siblings('.chat-textarea')[0],
+                        message;
+                    message = textarea.value;
+                    textarea.value = '';
+                    textarea.focus();
+                    if (message !== '') {
+                        this.onMessageSubmitted(message);
+                        _converse.emit('messageSend', message);
+                    }
+                    this.setChatState(_converse.ACTIVE);
+                },
+
+                render: function() {
+                    /* Render the UI container in which chat room messages will
+                     * appear.
+                     */
+                    this.__super__.render.apply(this, arguments);
+                    this.$('.sendXMPPMessage').append('<button type="button" class="pure-button send-button">Send</button>');
```

Comment:
Do you want me to do something like this?
``
this.$el.attr('id', this.model.get('box_id'))
                        .html(some_new_template(
                                _.extend(this.model.toJSON(), {
                                        show_toolbar: _converse.show_toolbar,
                                        show_textarea: true,
                                        title: this.model.get('fullname'),
                                        unread_msgs: __('You have unread messages'),
                                        info_close: __('Close this chat box'),
                                        label_personal_message: __('Personal message')
                                    }
                                )
                            )
                        );
``